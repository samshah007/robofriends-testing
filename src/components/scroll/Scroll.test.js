import { shallow } from "enzyme";
import React from "react";
import Scroll from './Scroll'

it('expect to render Card component', () => {
    expect(shallow(<Scroll></Scroll>).length).toEqual(1)
});
it('take snap of Card component', () => {
    expect(shallow(<Scroll />)).toMatchSnapshot();
});