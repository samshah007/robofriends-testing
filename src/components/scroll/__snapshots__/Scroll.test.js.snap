// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`take snap of Card component 1`] = `ShallowWrapper {}`;
